test/native.js

```diff
@@ -311,6 +312,62 @@ test('bigints', { skip: !hasBigInts }, function (t) {
 	t.end();
 });
 
+test('toPrimitive', { skip: !toPrimitive }, function (t) {
```

Comment:
this skip can never happen, because it’ll always be truthy

Reply:
```suggestion
test('toPrimitive', function (t) {
```